### PR TITLE
youtube-cleanup: fix remove-text-after-buttons-below-video hiding the Join button

### PR DESCRIPTION
**Pull request by BPower0036**
See https://www.youtube.com/watch?v=I0jeYe8iNWo
"Join"/"Become a member" is now disappeared

### CODE REVIEW

File: data/filters/youtube-cleanup.yaml [hunk 1]
```diff
@@ -46,8 +46,8 @@ template: |
   {{#if remove-text-after-buttons-below-video}}
   www.youtube.com###info > * [id^="top-level-buttons"] [class*="button-renderer"] [id="text"]:not(:has-text(/\d/))
   www.youtube.com##.ytd-watch-metadata > .top-level-buttons #text:not(:has-text(/[\d]/))
-  www.youtube.com##.ytd-watch-metadata ytd-button-renderer .yt-spec-button-shape-next .yt-spec-button-shape-next--button-text-content
-  www.youtube.com##.ytd-watch-metadata ytd-button-renderer .yt-spec-button-shape-next__icon:style(margin-right: -6px !important; margin-left: -6px !important;)
+  www.youtube.com###actions..ytd-watch-metadata ytd-button-renderer .yt-spec-button-shape-next .yt-spec-button-shape-next--button-text-content
```
[the remark this answers is not in the document]
Comment (BPower0036) [reply data/filters/youtube-cleanup.yaml:49]: Yes, it is a mistake

[the remark this answers is not in the document]
Comment (xvello) [reply data/filters/youtube-cleanup.yaml]: Thanks! Tested OK on the video you linked. You just need to update the test case.

I'd love to get your feedback on the contribution flow if you have some time in the future.

[the remark this answers is not in the document]
Comment (BPower0036) [reply data/filters/youtube-cleanup.yaml]: I was already wondering why the actions kept failing, but now it should be solved.

To get some feedback:
Contribution is easy. You only need to read carefully and don't forget things, like I just did.

**Review by xvello — APPROVED** [review]
LGTM, thanks!

File: data/filters/youtube-cleanup.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (xvello) [reply data/filters/youtube-cleanup.yaml]: > I was already wondering why the actions kept failing, but now it should be solved.

I pushed filter tests at the start of the CI, but I guess that the action logs are not easy to find (you have to click on the red cross icon, then the action). I'll check whether the action can comment on the PR with the results of the validation, for increased visibility.

[the remark this answers is not in the document]
Comment (xvello) [reply data/filters/youtube-cleanup.yaml]: > The only improvement you could make in the future is maybe adding a wiki

Indeed! Github wikis are pretty bad for this kind of projects though, as you need write permissions on the repo to edit the wiki, but I started [a markdown file](https://github.com/letsblockit/letsblockit/blob/main/data/filters/README.md) that we should flesh out.

[the remark this answers is not in the document]
Comment (BPower0036) [reply data/filters/youtube-cleanup.yaml]: I never knew that. Some how missed it. But it looks great and self-explanatory.

**Comment by xvello** [comment]
Thanks for your ongoing contributions @BPower0036.

I invited you to the @letsblockit/contributors team so that you can open branches on this repository instead of manually syncing your fork. I could you please send me an email so that I can contact you if needed?

**Pull request merged**